Dheeraj2407/Face-web
# Dockerized the app

Test Plan:
1. Build image
2. Create container
3. Start the container and verify that it runs without errors.

Result:
![image](https://user-images.githubusercontent.com/46856312/150629908-a7ec6400-50ec-4ff2-bf66-98e4a29e04f5.png)
![image](https://user-images.githubusercontent.com/46856312/150629913-792f7c83-1154-4a86-a23b-a64fe17ead95.png)
